### PR TITLE
Event Bus is the first component that should be started.

### PR DESCRIPTION
**Pull request by mookkiah**
### Description
Conflicting information on which component should be started first in fully distributed grid 4.

Question raised in slack channel about this - https://seleniumhq.slack.com/archives/C0ABCS03F/p1618484382029000

### Motivation and Context
Avoiding confusion on which component should be started and why.

### Types of changes
- [ x] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/grid/grid_4/components_of_a_grid.ja.md [hunk 1]
```diff
@@ -3,6 +3,12 @@ title: "グリッドのコンポーネント"
 weight: 1
 ---
 
+{{% notice info %}}
```
Comment (harsha509) [inline docs_source_files/content/grid/grid_4/components_of_a_grid.ja.md:6]: Hi @mookkiah ,

Could you please let me know why this is added ?

As the page is already translated, we've removed the message headers. If there is any change required in translated doc
1. Please replace the translated text with the text in english (or) entire paragraph in this case
2. Add the notice as it needs to be translated later

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: docs_source_files/content/grid/grid_4/components_of_a_grid.ja.md (hunk 1 above)
Comment (mookkiah) [reply docs_source_files/content/grid/grid_4/components_of_a_grid.ja.md:6]: It does makes sense to replace the changed section in english to help the translator to find it quickly.
I did it that way. @JS31096 please check.

**Review by harsha509 — APPROVED** [review]

**Review by harsha509 — APPROVED** [review]
Thank you @mookkiah !

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium docs @mookkiah 🎉  🎉

**Pull request merged**